src/hooks/useAccounts.tsx

```diff
@@ -348,11 +379,19 @@ const useAccountsContext = () => {
   // Disconnect wallet / accounts
   const disconnectLocalDydxWallet = () => {
     setLocalDydxWallet(undefined);
+    setLocalNobleWallet(undefined);
+    setLocalOsmosisWallet(undefined);
+    setLocalNeutronWallet(undefined);
```

Comment:
this was just straight up left out before.

I think eventually we make these functions so we derive on the fly as needed instead of storing within state.